bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -12,6 +12,7 @@ class PyNeurodamus(PythonPackage):
     git = "https://github.com/BlueBrain/neurodamus.git"
 
     version("develop", branch="main")
+    version("3.0a1", tag="3.0.0-alpha")
```

Comment:
@WeinaJi : just be aware of how you tag / name the version

```bash
$ spack-python

>>> from spack.version import Version as V;
>>> print(V('3.0') > V('3.0a1'))
False
>>> print(V('3.0.0') > V('3.0a1'))
True
>>> print(V('3.0.0') > V('3.0.a1'))
True
```

i.e. if you create `3.0` as next version in spack then it won't be greater than `3.0a`!

Reply:
here is our old Slack discussion: https://bluebrainproject.slack.com/archives/C023LSDKJUU/p1680617713058199?thread_ts=1680610388.183389&cid=C023LSDKJUU

Reply:
The next stable version will be 3.0.0. Why `3.0` ? We always use the versioning `x.y.z`.

Reply:
yes...just want to highlight this naming aspect!